.github/workflows/check-label-for-pr.yml

```
@@ -0,0 +1,27 @@
+          labels: |
+            kind/breaking-change
+            kind/feature
+            kind/feature-request
+            kind/bug
+            kind/other
+            kind/user-story
+            kind/dependencies
+            kind/enhancement
+            kind/incident
+            kind/ignore-for-release
+            kind/chore
```

yamllint

[error] no new line character at the end of file

(new-line-at-end-of-file)